src/devices/Ak8963/README.md

```diff
@@ -0,0 +1,115 @@
+# AK8963 - Magnetometer
+
+The AK8963 is a magnetometer that can be controlled either thru I2C either thru SPI. It is present in other sensors like the [MPU9250](../Mpu9250/README.md). This implementation fully supports the I2C mode and the usage thru the MPU9250. It does not support SPI.
```

Comment:
could this re-use the implementation in this case?

Reply:
I think it already does (just reviewing from top to bottom so didn't see it earlier)

Reply:
yes, for SPI, it is possible to reuse the implentation. Only the WriteRegister, ReadByte and ReadByteArry primitives will have to be adjusted. A little bit or rewor will be need but should be quite easy. As I don't have the full ak8963 and only the I2C implementation, I'm not able to test to thru I2C. All the rest of the code is fully agnostic.